pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.symphony.platformsolutions</groupId>
+    <artifactId>symphony-api-client-java-parent</artifactId>
+    <version>1.3.0-SNAPSHOT</version>
```

Comment:
Why the version is 1.3.0?

Reply:
Because I thought we had deliveries with version 1.2.x and wanted to increase. Indeed, latest deliveries are 1.2.x so let me change that.